pkg/el/google-fluentd-catch-all-config.spec

```diff
@@ -1,7 +1,7 @@
 Summary: Catch-all configuration files for google-fluentd on GCE.
 Name: google-fluentd-catch-all-config
 Version: %{package_version}
-Release: %{package_build_num}%{?dist}
+Release: %{package_build_num}
```

Comment:
Remind me again as to why the dist gets removed. I think I vaguely remember the logic on SuSE-specific naming conventions, but a quick 1 liner would be good.

Reply:
If we leave this in, the package will embed the distro it was built on (in our case, CentOS 7). Since the package is distro-independent, it shouldn't claim to be for a particular distro.